src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
     }
```

Comment:
You are calling 'Color.values()' multiple times which creates a new array every time it's called. To improve performance, you should call it once, store it in a variable, and then use that variable. For example:

Color[] colors = Color.values();
int index = random.nextInt(colors.length);
return colors[index];